addon/components/mdc-toolbar/title.js

```diff
@@ -1,16 +1,21 @@
 import Ember from 'ember';
 import layout from '../../templates/components/mdc-toolbar/title';
-import { MDCToolbarFoundation } from '@material/toolbar';
-
-const { strings } = MDCToolbarFoundation;
 
 export default Ember.Component.extend({
   //region Ember Hooks
   layout,
-  classNames: [`${strings.TITLE_SELECTOR}`],
+  classNames: ['mdc-toolbar__title'],
```

Comment:
:+1: Good add, but please import this class from the Foundation ([here](https://github.com/material-components/material-components-web/blob/21db79416a707cdc13d06c616f276ea5e4e73a3e/packages/mdc-toolbar/constants.js#L28)), as we do in `{{mdc-toolbar}}`.

https://github.com/secondstreet/ember-material-components-web/blob/bf27d26cac0cf141fae9c4561b8980a1186ef95e/addon/components/mdc-toolbar.js#L10

Reply:
@Kerrick are you sure? Currently, this line change fixes the bug. The `strings.TITLE_SELECTOR` value comes with a leading period.  In other components such as [here](https://github.com/secondstreet/ember-material-components-web/blob/master/addon/components/mdc-toolbar/section.js#L19) or [here](https://github.com/secondstreet/ember-material-components-web/blob/master/addon/components/mdc-toolbar/row.js#L9) we hard-coded the class name.

Reply:
Oh, good catch. It was in strings as a selector, not in cssClasses as a classname.